src/privileged/trackers/framescript.js

```diff
@@ -1,19 +1,13 @@
 /* global XPCOMUtils, sendAsyncMessage, docShell */
 const {classes: Cc, interfaces: Ci} = Components;
 const trackerListener = {
-  QueryInterface: XPCOMUtils.generateQI(["nsIWebProgressListener", "nsISupportsWeakReference"]),
+  QueryInterface: ChromeUtils.generateQI(["nsIWebProgressListener", "nsISupportsWeakReference"]),
   onSecurityChange: function(webProgress, request, state) {
-    const isBlocking = state & Ci.nsIWebProgressListener.STATE_BLOCKED_TRACKING_CONTENT;
-    const isAllowing = state & Ci.nsIWebProgressListener.STATE_LOADED_TRACKING_CONTENT;
-    if (isBlocking || isAllowing) {
-      // There are trackers on this page.
+    // This information should only be stored on the top-level docshell, so we use that even in frames.
```

Comment:
does this mean any trackers in an iframe are missed, or they will just be saved on the top level?

Reply:
Assuming this was implemented as specified (which it looks like) then this will just save all blocked trackers (even from sub-frames) on the top level document.